pyanaconda/installation.py

```diff
@@ -167,11 +168,6 @@ def _prepare_configuration(storage, payload, ksdata):
 
     configuration_queue.append(generate_initramfs)
 
-    # join a realm (if required)
-    join_realm = TaskQueue("Realm join", N_("Joining realm"))
```

Comment:
The task for joining the realm needs to run after setting up the network, but the tasks of the Security module are running before that. I guess that the best solution is to create a new DBus method for this task for now. Please, document this requirement in the docstring of `DiscoverRealmWithTask`.

Reply:
Indeed! I've split out the task geter to a separate & temporary DBUS method and moved its invocation to the original place where the kickstart execute method has been previously. That should hopefully fix things. :)